Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "bus_queue"
-version = "0.3.10"
+version = "0.4.0"
```

Comment:
Because the changes involved require that the sync::Publisher method broadcast use a mutable reference to self instead of an immutable one, the version is bumped into 0.4.0.